README.md

```diff
@@ -351,6 +351,13 @@ It's possible to modify the discovery ISO (via the API) to enable password login
 
 More information is available here: [Set discovery ISO user password example](docs/set-discovery-password.md)
 
+## Testing internal builds of OCS
+```shell
+export OCS_TEST_IMAGE=<custom build image>
```

Comment:
Can you please also add an example of whole url pointing to the build in quay.io? 
```
quay.io/rhceph-dev/ocs-registry:4.8.0-420.ci
```
This can be mentioned maybe after the shell block as example of: `<custom build image>`

Reply:
done